pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -153,9 +153,13 @@ type ClusterSpec struct {
 	// Optional: ImagePullSecret references a secret with container registry credentials. This is passed to the machine-controller which sets the registry credentials on node level.
 	ImagePullSecret *corev1.SecretReference `json:"imagePullSecret,omitempty"`
 
+	// Optional: CNIPluginSettings contains the spec of the CNI plugin used by the Cluster.
 	CNIPlugin *CNIPluginSettings `json:"cniPlugin,omitempty"`
 
-	ClusterNetwork  ClusterNetworkingConfig   `json:"clusterNetwork"`
+	// Optional: ClusterNetworkingConfig specifies the different networking parameters for a cluster.
+	ClusterNetwork ClusterNetworkingConfig `json:"clusterNetwork"`
+
+	// Optional: MachineNetworkingConfig specifies the networking parameters used for IPAM.
```

Comment:
Same here. Should be the field name and maybe more descriptive to be honest:

```suggestion
	// Optional: MachineNetworks specifies a list of networking parameters used for IPAM.
```